### PR TITLE
Feature/mango loan state

### PR DESCRIPTION
**Pull request by Chepelau**

### CODE REVIEW

File: src/loans/mango.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+                elif position < 0:
+                    self.loan_entities[mango_account].debt.set_value(token=mint_address, value=abs(position))
+
+                else:
+                    continue
+        self.last_slot = int(time.time())
+                
+
+
+def liq_debt(x: pd.Series, collateral_collumn: str, debt_collumn: str) -> float:
```
Comment (lukaspetrasek) [inline src/loans/mango.py:160]:
```suggestion
def liq_debt(x: pandas.Series, collateral_collumn: str, debt_collumn: str) -> float:
```

**Comment by DaveVodrazka** [comment]
LGTM

File: src/loans/mango.py (hunk 1 above)
Comment (Chepelau) [reply src/loans/mango.py:160]: no

**Comment by Chepelau** [comment]
Merging.

**Pull request merged**